adagios/rest/tests.py

```diff
@@ -27,8 +27,24 @@
 from django.utils.translation import ugettext as _
 import json
 
+import adagios.utils
+
 
 class LiveStatusTestCase(unittest.TestCase):
+
+    def setUp(self):
```

Comment:
Would be nice to have a simple setup and terminate for the fake env. Seems like we are copying this all over the place. Other then that, good stuff.